packages/importer/src/importer.ts

```diff
@@ -161,32 +164,38 @@ export class OneSchemaImporterClass extends EventEmitter {
       message.messageType = "init-session"
       message.sessionToken = mergedParams.sessionToken
     } else {
-      message.messageType = "init"
       message.userJwt = mergedParams.userJwt
```

Comment:
idk if this makes it hard to read:

both `"init-template-group` and `"init"` need userJWT and import config... but maybe its best to just duplicate that